src/app/api/marketplace/route.js

```
@@ -0,0 +1,303 @@
+    // Add sorting
+    switch (sortBy) {
+      case 'price-low':
+        constraints.push(orderBy('price', 'asc'));
+        break;
+      case 'price-high':
+        constraints.push(orderBy('price', 'desc'));
+        break;
+      case 'name':
+        constraints.push(orderBy('name', 'asc'));
+        break;
+      case 'newest':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)